webapp/src/components/ImageEditor/ImageCanvas.tsx

```diff
@@ -95,6 +95,9 @@ class ImageCanvasImpl extends React.Component<ImageCanvasProps, {}> implements G
     }
 
     componentDidMount() {
+        // move initial focus off of the blockly surface
+        (document.activeElement as HTMLElement)?.blur();
```

Comment:
@jwunderl is this okay for accessibility?

Reply:
I think it's better than the current state (currently focus is being left on the svg of the preview image they clicked on)

That said definitely some accessibility work in general for image editor - right now it doesn't capture focus like a modal typically would (which is presumably why this is here in the first place, to get focus off hidden elements), and only the size inputs have tabstops - there's actually no obvious way to escape once you've entered the image editor keyboard, as the done button doesn't have a tabstop.